domains/gamingklex.json

```
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "GamingKlex",
+        "email": "gamingklexmail@gmail.com"
+      },
+      "record": {
+        "TXT": "test"
```

Please use a valid record. Something like:
```suggestion
        "CNAME": "gamingklex.github.io"
```